Translations/Polish/README.md

```diff
@@ -192,7 +192,12 @@ $(".foo div#bar:eq(0)")
 
 * Jaka jest różnica między klasami oraz idendyfikatorami w CSS?
   Idendyfikatory są unikalne, każdy element może posiadać tylko jeden identyfikator, dodatkowo przeglądarka może używać identyfikatorów do nawigacji (scrollując do elementu z danym ID). Klasy nie są unikalne oraz elementy mogą posiadać więcej niż jedną klasę, nie mają większego znaczenia dla samej przeglądarki.
+* Opisz BFC(Block Formatting Context) i powiedz jak działa.
+  A block formatting context is a part of a visual CSS rendering of a Web page. It is the region in which the layout of block boxes occurs and in which floats interact with each other. The rules for positioning and clearing of floats in apply only to things within the same block formatting context.
```

Comment:
To nie do końca oddaje postać rzeczy
https://www.w3.org/TR/CSS21/visuren.html#block-formatting
https://www.sitepoint.com/understanding-block-formatting-contexts-in-css/